sphinx/transforms/i18n.py

```diff
@@ -327,26 +326,6 @@ def is_refnamed_ref(node):
 
                 self.document.note_refname(new)
 
-            # refnamed footnote and citation should use original 'ids'.
```

Comment:
To change the warning message, please make a copy of this block to separate verifications of foonote_refs and citation_refs.